.pytool/Plugin/ImageValidation/ReadMe.md

```diff
@@ -7,13 +7,64 @@ file path is provided via the command line as `PE_VALIDATION_PATH=<PATH>` or
 can be configured in the the PlatformBuild.py within the `SetPlatformEnv()`
 method using 
 `self.env.SetValue("PE_VALIDATION_PATH", <PATH>, "Platform Hardcoded")`. A 
-profile is equivalent to the file types defined in the platform's fdf. All
-profiles must be defined, forcing the developer to acklowedge each, however
+profile is equivalent to the binary's MODULE_TYPE as defined in it's inf. All
+profiles must be defined, forcing the developer to acknowledge each, however
 requirements for each profile do not need to be specified... If one or more
 requirement does not exist, the "DEFAULT" profile requirements will be used.
 The developer can have default requirements via the "DEFAULT" profile then
 override those requirements in other profiles. An example of a full config
-file can be seen at the bootom of the readme.
+file can be seen at the bottom of the readme.
+
+A default set of requirements are provided by the tool that apply to all
+X64 and AARCH64 binaries. These requirements are:
+
+1. IMAGE_BASE = 0x0
+2. 4k Section Alignment (X64, AARCH64), 64k Section Alignment (AARCH64-DXE_RUNTIME_DRIVER)
+3. No Section can be both Write and Execute.
+
+Image Base is verified to be 0x0 for EFIs as some PE loaders (including edk2's)
+will attempt to load the image at that address, which is generally not needed in a
+UEFI environment. Section alignment is verified at 4k as it is required for DXE
+memory protections, exclduing AARCH64 DXE_RUNTIME_DRIVER binaries, which are
+required to be 64K per the UEFI specifcation. Sections are verified **not** to be
+both Write and Execute as it is required by DXE memory protections.
+
+These requirements can be expanded to IA32, ARM, etc, or overwritten for X64
+and AARCH64 as defined in the configuration file provided via PE_VALIDATION_PATH.
+Individual profiles for individual architectures can be overwritten in this file.
+
+Configuration supports poth `json` and `yaml` formats. Here is a small example for setting
+an ignore list, and overriding configuration for a specific module type. Read below for all
+customization options.
+
+```json
+{
+    "IGNORE_LIST": ["Driver.efi"],
+    "X64": {
+        "DXE_DRIVER": {
+            ...
+        }
+    } 
+}
+```
+
+```yaml
+IGNORE_LIST:
+    - Driver.efi
+X64:
+    DXE_DRIVER:
+        ...
+
+```
+
+**See Example below for customizing Image Validation**
+
+## Ignoring Files
+
+If your platform deems a particular binary does not, and cannot meet the
+requirements set by the Image Validation plugin, or the platform's custom
+config, it can be ignored by adding a `IGNORE_LIST = [...]` section to the
+josn file provided via PE_VALIDATION_PATH.
```

Comment:
```suggestion
JSON file provided via PE_VALIDATION_PATH.
```

Reply:
JSON is not even correct anymore - supports yaml and json. Will update. Good catch.

Reply:
Updated, thanks